kvmagent/kvmagent/test/http/conftest.py

```
@@ -0,0 +1,743 @@
+    http_client.register_async_uri(plugin.FACT_PATH, plugin.fact)
+
+    if not http_client._started:
+        http_client.start()
+
+    # In remote mode, verify the bootstrap connect succeeded
+    if http_client.is_remote:
+        try:
+            rsp = http_client.post_sync('/host/echo/', {})
+            if not getattr(rsp, 'success', False):
+                logger.warning('remote echo failed: %s', getattr(rsp, 'error', ''))
+        except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)